src/Analyzer.elm

```diff
@@ -141,6 +144,10 @@ initialContext functions =
         |> Rule.withModuleNameLookupTable
 
 
+
+-- Keep track of top level function
```

Comment:
Can we use the documentation function comment style that elm-format likes here?

```{-| .... -}```

Reply:
Ah, I never noticed it did that